### ISSUE TITLE
Ordering low->high zoom sometimes unfavorable

### ISSUE THREAD

**Issue by jojo4u** [matkoniecz/CartoCSSHelper#31]
I had trouble locating the turning cycles in https://github.com/gravitystorm/openstreetmap-carto/pull/2165 For every example I had to scroll down, locating the circle and scrolling up again.

The more natural ordering of the rendering examples would be high zoom panes on top and low zoom on bottom.
Example: https://cloud.githubusercontent.com/assets/899988/16024603/25126fb8-31c3-11e6-82b3-3711742b85ef.png

**Comment by matkoniecz** [comment]
Yes, it is a good idea. It makes no change for me (as my stupid picture viewer goes to a middle of the screen*) and will be helpful for people using smarter image browsers.

*in case that somebody is aware about image viewer for inux capable of displaying top of file rather going to the middle and also capable of [properly deleting files](http://askubuntu.com/questions/551066/simple-image-viewer-that-is-using-trash-rather-than-permanently-deleting-files) or at least [pretending to do this](http://askubuntu.com/questions/551295/image-viewer-with-simple-moving-to-a-definable-folder?lq=1) I would be thankful.
